docs/nodes/nodes-library/nodes/GoogleCalendar/README.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /nodes/n8n-nodes-base.googleCalendar
+---
+
+# Google Calendar
+
+[Google Calendar](https://www.google.com/calendar/) is a time-management and scheduling calendar service developed by Google.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Google/README.md).
+:::
+
+## Basic Operations
+
+- Event
+    - Add a event to the calendar
```

Comment:
You'll have to remove the 'the' here.

<img width="243" alt="Screenshot 2020-07-01 at 09 15 28" src="https://user-images.githubusercontent.com/7481165/86214413-7403d080-bb7b-11ea-8d49-f295887f24c2.png">

Reply:
okay done!